src/post-submission-actions/program-enrollment-action.ts

```diff
@@ -33,26 +36,32 @@ export const ProgramEnrollmentSubmissionAction: PostSubmissionAction = {
             (enrollment) => enrollment.program.uuid === programUuid && enrollment.dateCompleted === null,
           );
           if (hasActiveEnrollment) {
-            throw new Error('Cannot enroll patient to program. Patient already has an active enrollment');
+            showToast({
+              title: t('errorSavingEnrollment', 'Error saving enrollment'),
+              kind: 'error',
+              critical: false,
+              description: t('cannotEnrollPatientToProgram', 'Cannot enroll patient to program. Patient already has an active enrollment'),
+            });
           }
+          return;
         }
         createProgramEnrollment(payload, abortController).then(
           (response) => {
             if (response.status === 201) {
               showToast({
                 critical: true,
                 kind: 'success',
-                description: 'It is now visible in the Programs table',
-                title: 'Program enrollment saved',
+                description: t('nowVisibleInProgramsTable', 'Changes to the program are now visible in the Programs table'),
```

Comment:
Nit: I think the previous message reads better for this create context. I wouldn't expect the same message here and in the edit context.